python/pyspark/pandas/indexing.py

```diff
@@ -1146,9 +1146,13 @@ def _select_rows_else(
         )
 
     def _get_from_multiindex_column(
-        self, key: Optional[Tuple], missing_keys: Optional[List[Tuple]],
-            labels: Optional[List[Tuple]] = None, recursed: int = 0
-    ) -> Tuple[List[Tuple], Optional[List[spark.Column]], Any, bool, Optional[Tuple]]:
+        self,
+        key: Optional[Tuple],
+        missing_keys: Optional[List[Tuple]],
+        labels: Optional[List[Tuple]] = None,
+        recursed: int = 0
+    ) -> Tuple[List[Tuple], Optional[List[spark.Column]], Any, bool,
+               Optional[Tuple]]:
```

Comment:
nit: Would it be better to put `Optional[Tuple]]` to its previous line?
`Tuple[List[Tuple], Optional[List[spark.Column]], Any, bool, Optional[Tuple]]:`

Reply:
Sure. Thanks for pointing that out.
I will pay more attention on coding style next time

Reply:
Thank you.